pop-api/examples/fungibles/README.md

```diff
@@ -0,0 +1,41 @@
+# PSP22 Fungible Token with Pop API
+
+
+This [ink!][ink] contract shows a contract that allows interaction and management of a fungible token following the [PSP22 standard][psp22], utilizing the [Pop API Fungibles][pop-api-fungibles] feature. In this specific contract only the contract owner has a permission to call specific methods.
+
+## Design Goals
+
+- Token exists as an asset on the Pop Network, rather than being confined solely to the contract.
+- Contract is the `origin` of the calls made by `pop-api`.
+- Only the contract owner has a permission to call specific methods.
+
+[Learn more how Pop API works.](/pop-api/README.md)
+
+## Test with Pop DR!nk
+
+Because the contract uses `pop-api` which calls to the runtime, it requires a special crate called [pop-drink][pop-drink] to test the contract. See how the contract is tested in [tests](./tests.rs).
```

Comment:
```suggestion
## Test with Pop Drink

Since this contract interacts directly with Pop’s runtime through the Pop API, it requires [Pop Drink](https://github.com/r0gue-io/pop-drink) for testing. See how the contract is tested in [tests](./tests.rs).
```